src/func.rs

```diff
@@ -293,16 +293,23 @@ impl<'args> FuncInvocation<'args> {
         return_val: Option<RuntimeValue>,
         externals: &'externals mut E,
     ) -> Result<Option<RuntimeValue>, ResumableError> {
-        match self.kind {
-            FuncInvocationKind::Internal(ref mut interpreter) => {
-                if !interpreter.state().is_resumable() {
-                    return Err(ResumableError::AlreadyStarted);
+        use crate::TrapKind;
```

Comment:
Oh, this is a new keyword and apparently it doesn't require the 2018 edition. What is the version of a compiler is required in order to use this keyword?

Reply:
https://github.com/rust-lang/rust/issues/45477